src/Html.php

```diff
@@ -1733,7 +1733,11 @@ public static function addCssClass(array &$options, BackedEnum|array|string|null
                 /** @psalm-var string[] $options['class'] */
                 $options['class'] = self::mergeCssClasses($options['class'], (array) $class);
             } else {
-                /** @psalm-var string $options['class'] */
+                /**
+                 * @psalm-var string $options['class']
+                 * @var string[] $classes We assume that `$options['class']` is valid UTF-8 string, so `preg_split()`
```

Comment:
Makes sense to mention this in the method doc block.

Reply:
Done